events/elections/2020/davanumsrinivas.md

```diff
@@ -0,0 +1,37 @@
+# Davanum Srinivas
+
+You may know me as [@dims] on github/slack/twitter.
+
+## How I contribute / What have i done
+
+Help nudge things along in Issues and PRs in the main k/k repository. Served on the Steering Committee since 2018 (one 2 year tenure).
+
+Filed issues, worked on features, fixed bugs as well in Kubernetes.
+
+Regularly participate in various cross cutting SIGs like Architecture, Contrib-Ex, Testing, Release etc. Currently co-chair of SIG-Arch, WG-k8s-Infra.
+
+For various releases i have helped with triaging issues, maintain milestones for PRs and Issues, dig into CI failures etc.
+
+I like to research old outstanding issues that bother a lot of users and try to come up solutions.
+
+Help with things that hinder the contribution process for folks who do not work full time on kubernetes and try to pitch in to make it better for them.
+
+Specific items i have helped drive: 
+- run code-organization sub-project for SIG-Arch. Help with Conformance related discussions as well.
+- breaking up things like klog and openstack cloud provider code into a separate repository with their own CI.
+- bootstrapping the cluster-api-provider for openstack
+- enabling multi-arch support for conformance tests and release artifacts.
+
+## What i would like to do
+
+Having been part of various Open Source projects and foundations i would like to ensure that Kubernetes will be vibrant and resilient over the ups and downs of the hype curve.
+
+We are currently optimized for large-ish teams with full time contributors and i am keen on improving participation of individuals who are occasional contributors but with real world needs.
+
+Build better bridges with the broader ecosystems and other open source projects and teach/learn from each other.  
+
+## Where I work
+
+VMware
+
+[@dims]: https://github.com/dims
```

Comment:
i was going to add mine , when i saw your PR https://github.com/kubernetes/community/blob/master/events/elections/2020/biotemplate.md .  Its different from what is in the template or is that template in the process of being updated ?